octobot_trading/channels/orders.py

```diff
@@ -62,12 +64,13 @@ async def perform(self, orders, is_closed=False, is_from_bot=True):
                                         is_closed=parse_is_closed(order),
                                         is_updated=changed)
 
-            if symbol is not None:
-                possible_new_order = await self._check_missing_open_orders(symbol, orders) or possible_new_order
-            if possible_new_order:
-                # possibility new order: refresh portfolio to ensure available funds are up to date
-                await get_chan(BALANCE_CHANNEL, self.channel.exchange_manager.id).get_internal_producer(). \
-                    refresh_real_trader_portfolio()
+            if self.SHOULD_CHECK_MISSING_OPEN_ORDERS:
+                if symbol is not None:
+                    possible_new_order = await self._check_missing_open_orders(symbol, orders) or possible_new_order
+                if possible_new_order:
```

Comment:
this part (pf update) should be processed even if `SHOULD_CHECK_MISSING_OPEN_ORDERS` is false: `possible_new_order` can be true due to line 57